uw-frame-components/portal/widgets/controllers.js

```diff
@@ -149,7 +149,21 @@ define(['angular'], function(angular) {
      */
     $scope.getPrettyDate = function(dateString) {
       // Create a new date if a date string was provided, otherwise return null
-      return dateString ? new Date(dateString) : null;
+      if (dateString === undefined) {
```

Comment:
Eh, yeah. It should probably just be reverted back to the old ternary. I made the change before I was planning on changing rssToJson :P